### PR TITLE
JR2-415 Add slider input to job search filter

### PR DESCRIPTION
**Pull request by varusakov**

### CODE REVIEW

File: src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.html [hunk 1]
```diff
@@ -28,4 +28,11 @@
         <label class="small">{{'job-search.filter.working-time.label' | translate}}</label>
         <jr2-range-input formControlName="workingTime" [min]="0" [max]="100" [step]="10"></jr2-range-input>
     </div>
+    <div class="form-group range-slider">
+        <label class="small">{{ 'job-search.filter.online-since.label' | translate}}</label>
+        <nouislider [min]="1" [max]="60" [config]="{step: 1}" class="range-input"
+                    [formControl]="filterForm.get('onlineSince')"></nouislider>
```
Comment (Invizible) [inline src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.html]: can we specify just formControlName?

Comment (varusakov) [reply src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.html]: Need to check that

Comment (birom) [reply src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.html]: Have you already checked this?

Comment (varusakov) [reply src/main/webapp/app/job-search/job-search-sidebar/job-search-filter/job-search-filter.component.html]: No, it's not possible with this component. `formControlName="onlineSince"` binding causes error

**Pull request merged**